.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,37 +5,24 @@ Summarize your PR. If it involves visual changes include a screenshot or gif.
 - [ ] If a plugin configuration key changed, check if it needs to be allowlisted in the cloud and added to the [docker list](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)
-- [ ] This renders correctly on smaller devices using a responsive layout. (You can test this [in your browser](https://www.browserstack.com/guide/responsive-testing-on-local-server))
-- [ ] This was checked for [cross-browser compatibility](https://www.elastic.co/support/matrix#matrix_browsers)
-
-
-### Risk Matrix
-
-Delete this section if it is not applicable to this PR.
+- [ ] This was checked for breaking API changes and was [labeled appropriately](https://www.elastic.co/guide/en/kibana/master/contributing.html#_add_your_labels)
+- [ ] This should appear in the **Release Notes** and follow the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)
 
-Before closing this PR, invite QA, stakeholders, and other developers to identify risks that should be tested prior to the change/feature release.
+### Identify risks
 
-When forming the risk matrix, consider some of the following examples and how they may potentially impact the change:
+Does this PR introduce any risks? For example, consider risks like hard to test bugs, performance regression, potential of data loss.
 
-| Risk                      | Probability | Severity | Mitigation/Notes        |
-|---------------------------|-------------|----------|-------------------------|
-| Multiple Spaces&mdash;unexpected behavior in non-default Kibana Space. | Low | High | Integration tests will verify that all features are still supported in non-default Kibana Space and when user switches between spaces. |
-| Multiple nodes&mdash;Elasticsearch polling might have race conditions when multiple Kibana nodes are polling for the same tasks. | High | Low | Tasks are idempotent, so executing them multiple times will not result in logical error, but will degrade performance. To test for this case we add plenty of unit tests around this logic and document manual testing procedure. |
-| Code should gracefully handle cases when feature X or plugin Y are disabled. | Medium | High | Unit tests will verify that any feature flag or plugin combination still results in our service operational. |
-| [See more potential risk examples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx) |
+Describe the risk, its severity, and mitigation for each identified risk. Invite stakeholders and evaluate how to proceed before merging.
 
+- [ ] [See some risk examples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx)
+- [ ] ...
 
-### For maintainers
```

Comment:
moved this up. Reviewers should verify all the checks are satisfied anyway before approving.